### ISSUE TITLE
Unable to log in

### ISSUE THREAD

**Issue by tfer6** [thor0215/hassio-xfinity-usage#64]
It worked occasionally with previous versions, but would eventually hit what I'm presuming was bot detection. The last week or so I've had no luck.  I tried the upgrade to 0.1.2.2 and still get the "Navigated to username page for the 2nd time." or "Navigated to password page for the  2nd time." error.  Logs attached.

I do NOT have 2FA on the account and I'm able to log in to the account with the configured ID and password.

[xfinitylog.txt](https://github.com/user-attachments/files/18202689/xfinitylog.txt)

**Comment by izzletodasmizzle** [comment]
Yes, same thing for me. Updated to 1.2.2 this morning, worked once or twice, and unfortunately has reverted back to the login issue, either username or password like you describe.
This might be a cat and mouse game that the developer unfortunately won't be able to keep up with. But I appreciate the effort!

**Comment by tankdeer** [comment]
Same here. And not sure if related,  but I frequently find the add-on has stopped. Even with watchdog enabled

**Comment by dschall** [comment]
Same for me, logs attached.
[15d584b4_xfinity-usage_2024-12-20T02-09-19.789Z.log](https://github.com/user-attachments/files/18205140/15d584b4_xfinity-usage_2024-12-20T02-09-19.789Z.log)

I just started trying this out, IIRC with version 0.1.1 and it didn't work from the start.
FWIW, I do have two accounts (one cancelled and one active) under that username, not sure if that confuses the plugin.

**Comment by J-CMartin** [comment]
I have had the same issues as reported above where it would work once and then stop, but if I restarted later sometimes it would work sometimes it would not. I realized that waiting longer was the key and I'm pleased to inform that I am having good luck with the setting to update every 12hours. 

**I recommend all to change the Pooling Time setting (Seconds) to 43200** and see if at least it will keep running that way!
